geoh5py/objects/drillhole.py

```diff
@@ -490,15 +490,23 @@ def add_vertices(self, xyz):
 
         return indices.astype("uint32")
 
-    def validate_interval_data(
+    def validate_interval_data(  # pylint: disable=too-many-locals
         self,
         from_to: np.ndarray | list,
         values: np.ndarray,
         collocation_distance: float = 1e-4,
-    ):
+        type_: str | None = None,

```

Comment:
Would be great if we could avoid another parameter. Any chance you can figure this out by checking the dtype of values?

Reply:
Good idea. So simple I don't even thought about it ;-)